spring-data-rest-webmvc/src/main/java/org/springframework/data/rest/webmvc/mapping/AssociationLinks.java

```diff
@@ -99,4 +103,31 @@ public boolean isLinkableAssociation(PersistentProperty<?> property) {
 		metadata = mappings.getMappingFor(property.getActualType());
 		return metadata == null ? false : metadata.isExported();
 	}
+
+	/**
+	 * Returns whether the given property is an association that should be exported.
+	 *
+	 * @param property can be {@literal null}
+	 * @return
+	 */
+	public boolean isExportableAssociation(PersistentProperty<?> property) {
+
+		// If the property is null or not an association, it's not exportable
+		if (property == null || !property.isAssociation()) {
```

Comment:
I think we should abstract this because using `JsonIgnore` is just one way to exclude properties from mappings. I think a `RepresentationMetadata` interface with an implementation that uses a `BasicClassIntrospector` to actually use the Jackson metamodel API to discover Jackson managed properties. See `DomainObjectReader.getJacksonProperties(…)` for inspiration.